packages/desktop-client/src/components/settings/Global.tsx

```diff
@@ -58,10 +65,7 @@ export default function GlobalSettings() {
           borderRadius: 4,
           overflow: 'auto',
           whiteSpace: 'nowrap',
-          // TODO: When we update electron, we should be able to
-          // remove this. In previous versions of Chrome, once the
-          // scrollbar appears it never goes away
-          '::-webkit-scrollbar': { display: 'none' },
+          ...webkitScrollbarStyle,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this change necessary?